tests/cluster-pipeline.test.sh

```
@@ -0,0 +1,1183 @@
+else
+  FAIL=$((FAIL+1))
+fi
+
+# ============================================================================
+# P2 — Body cap 64 KiB (constraints.md T1, Phase 1 Fetch).
+# ============================================================================
+#
+# Generate a 100 KiB body, build a JSON issues array, run the body-cap jq line
+# verbatim from SKILL.md Phase 1, assert capped body is <= 65536 bytes.
+echo "== P2 body cap 64 KiB"
+P2_BIG_BODY="$(python3 -c 'print("x"*102400, end="")')"
```

Shellcheck (0.11.0)

[warning] P2_BIG_BODY appears unused. Verify use (or export if used externally).

(SC2034)